### ISSUE TITLE
Request form: Required fields are marked as required

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2698]
## Issue description

As a user who relies on accessibility features,
I want all required fields in the request form to be clearly indicated as required,
so that I can accurately and efficiently complete the form without confusion or errors, in compliance with Section 508 accessibility guidelines.

## Acceptance Criteria

- [ ] Review the request form audit created as a part of #2866 
- [ ] Update the **Type of organization** page design to ensure that all required fields are clearly marked as required.
 - [ ] Update the **Federal Government branch** page design to ensure that all required fields are clearly marked as required.
- [ ] Update the **Election office** page design to ensure that all required fields are clearly marked as required.
- [ ] Update the **Purpose of your domain** page design to ensure that all required fields are clearly marked as required.
- [ ] Update the **Other employees from your organization** page design to ensure that all required fields are clearly marked as required.
- [ ] Update the **Requirements for operating a .gov domain** page design to ensure that all required fields are clearly marked as required.
- [ ] Update the **.gov domain** page design to ensure that all required fields are clearly marked as required.
- [ ] The updated designs pass the Trusted Tester success criteria for 5.B Descriptive Label
- [ ] Create a dev ticket to implement the changes.

## Additional Context

The request form has a few required fields without indicators that they are required. This fails the 508 [trusted tester success criteria label descriptive](https://github.com/Section508Coordinators/TrustedTester5.1/blob/main/Documents/Trusted%20Tester%20Test%20Process%20v5.1.3%20w%20Updates%20Highlighted.pdf). 

Gaby mentioned these fields were not marked as required intentionally because of USWDS guidance. [Here it is](https://designsystem.digital.gov/templates/form-templates/):

> Note: One-field forms are the exception and do not need to be marked with a required label or symbol.

This contradicts the section 508 guidelines but also does not apply to the request form because it has more than one field even though some pages only have one.

### Required fields
- Type of organization page: What kind of U.S.-based government organization do you represent?
-  Election office: Is your organization an election office?
- Federal government branch page: Which federal branch is your organization in?
  - _Note: The g in .gov should be capitalized according to our content guide. Try to sneak in that update if you can._

### References for creating a dev ticket with accessibility requirements in the A/C
- #2871 
- #2849 

## Issue Links
- Contains [Domain request: .gov domain step form field needs visual indicator that field is required](https://github.com/cisagov/manage.get.gov/issues/2357)
- Relates to Request form labels communicate required data format #2699
- Relates to [Make the "x" in search bars an accessible button #301](https://github.com/cisagov/get.gov/issues/301)
- The .gov domain page also needs a visual indicator but that will be addressed in #2699

**Comment by AnnaGingle** [comment]
Duplicate of #2698. Closing

**Comment by AnnaGingle** [comment]
Just kidding! I thought I made a new ticket, but I actually updated the existing one 😎

**Comment by PaulKuykendall** [comment]
Moved from parking lot to ready because it was in advertently closed earlier. This ticket is active and valid.
